docs/howto/openstack/neutron/new-network-control-panel.md

```diff
@@ -0,0 +1,57 @@
+# Creating a new network from the Cleura Cloud Control Panel
+
+Before creating a server in {{extra.brand}} Cloud, you need to have a 
+network to make the new server a member of. You may have more than one 
+network per region, so let us now walk through creating a new network 
+using the {{extra.brand}} Cloud Control Panel. Fire up your favorite web 
```

Comment:
We can use {{config.extra.gui}} value here instead, which is "Cleura Cloud Control Panel".

Just a heads-up: For some reason the config variables don't work when used in page title, so leave line 1 as it is. :)